quirks/websites-with-shared-credential-backends.json

```diff
@@ -198,5 +199,13 @@
     [
         "lookmark.io",
         "lookmark.link"
+    ],
+    [
+        "gap.com",
+        "bananarepublic.com",
+        "athleta.com",
+        "hillcity.com",
+        "oldnavy.com"
     ]
+
```

Comment:
Let’s remove this errant newline, please!

Reply:
I logged in to gap.com using my credential and then verified through their listed partner sites and it all worked on SSO methodology using a single account.

Reply:
More details from GAP sign in page -  https://secure-www.gap.com/profile/sign_in.do

<img width="650" alt="image" src="https://user-images.githubusercontent.com/3323086/83952407-93624480-a7ed-11ea-926c-6e9cc3b5cc09.png">